src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/ApplyBQSRIntegrationTest.java

```diff
@@ -90,18 +96,82 @@ public void testApplyBQSR(ABQSRTest params) throws IOException {
         args.add(new File(params.bam).getAbsolutePath());
         args.add("--" + StandardArgumentDefinitions.BQSR_TABLE_LONG_NAME);
         args.add(new File(resourceDir + "HiSeq.20mb.1RG.table.gz").getAbsolutePath());
-        args.add("-O"); args.add(outFile.getAbsolutePath());
+        args.add("-O");
+        args.add(outFile.getAbsolutePath());
         if (params.reference != null) {
             refFile = new File(params.reference);
-            args.add("-R"); args.add(refFile.getAbsolutePath());
+            args.add("-R");
+            args.add(refFile.getAbsolutePath());
+            if (params.args != null) {
+                Stream.of(params.args).forEach(arg -> args.add(arg));
+            }
+
+            runCommandLine(args);
+
+            SamAssertionUtils.assertSamsEqual(outFile, new File(params.expectedFile), refFile);
         }
-        if (params.args != null) {
-            Stream.of(params.args).forEach(arg -> args.add(arg));
+    }
+
+    @Test(dataProvider = "ApplyBQSRTest")
+    public void testApplyBQSRPath(ABQSRTest params) throws IOException {
```

Comment:
This test doesn't need to be run on the full `ApplyBQSRTest` `DataProvider` -- can you create a separate `DataProvider` with ONE of the test cases from the `ApplyBQSRTest` provider, and use that here?